extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -79,6 +92,22 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
       },
       true /** callToInitialize */
     );
+
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_ID,
+      (id) => {
+        this.setShoppingTagActive(id === this.pageEl_.id);
+      },
+      true /** callToInitialize */
+    );
+  }
+
+  /**
+   * Activate or deactivate shopping tag
+   * @param {boolean} isActive
+   */
+  setShoppingTagActive(isActive) {
+    toggleAttribute(this.shoppingTagEl_, 'active', isActive);
```

Comment:
I think you should be able to remove all this JS and it'd still work, using the `[active]` attr on the `amp-story-page`